truss/templates/control/control/helpers/inference_server_process_controller.py

```diff
@@ -1,18 +1,26 @@
+import logging
 import os
+import signal
 import subprocess
 
 from helpers.context_managers import current_directory
 
 
 class InferenceServerProcessController:
+
+    _inference_server_process: subprocess.Popen = None
+    _inference_server_port: int
+    _inference_server_home: str
+    _app_logger: logging.Logger
+    _inference_server_process_args: list[str]
+
     def __init__(
         self,
-        inference_server_home,
-        inference_server_process_args,
-        inference_server_port,
-        app_logger,
+        inference_server_home: str,
```

Comment:
thanks for adding types !!

Reply:
💯 ! Trying to add where it makes sense in the context of the change to start a bit of momentum in doing it